templates/web/bromley/reports/_list-entry.html

```diff
@@ -0,0 +1,6 @@
+<li class="[% c.cobrand.pin_colour(problem) %]">
```

Comment:
I note that this (and similar elsewhere it's used) has much less information than the upstream template – it loses distance away, last updated (which on all reports page can be quite important), state and the report's image. The last updated and image seem like they may be wanted, perhaps?